gemini/checkbox.gemini.js

```diff
@@ -37,8 +37,9 @@ gemini.suite('checkbox', function(root) {
                     .capture('hovered', function(actions) {
                         actions.mouseMove(this.checkbox);
                     })
-                    .capture('focused-hard', function(actions) {
-                        actions.focus(this.checkbox);
+                    .capture('focused-hard', function(actions, find) {
+                        // NOTE: can't focus on `<label>`, so find `checkbox__control` explicitly
+                        actions.focus(find(element + '__control'));
```

Comment:
@sipayRT @tadatuta i've fixed `gemini` tests to use `checkbox` instead of `checkbox__control`. The only problem is with with `focus()` element. But I'm sure, it's a gemini bug (gemini-testing/gemini#172)